ext_mongodb.php

```diff
@@ -746,7 +868,7 @@ public function getCode() : string
 
 	public function getScope() : mixed
 	{
-		if ( $this->scope !== NULL )
+		if ( isset( $this->scope) && $this->scope !== NULL )
```

Comment:
I agree, but then __set_state would *also* end up having "scope", and it wouldn't have passed the tests you wrote. ;-)